src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReportingController.kt

```diff
@@ -28,4 +28,10 @@ class ReportingController(
     reportingService.generateServiceProviderPerformanceReport(dateInterval.fromDate, dateInterval.toDate, userMapper.fromToken(authentication))
     return ResponseEntity.accepted().build()
   }
+
```

Comment:
commented out for now. Can be used to run the report currently.

Reply:
can we just remove this from this PR if its unused?